diff --git a/node_modules/vitepress/dist/client/theme-default/components/HoverImage.vue b/node_modules/vitepress/dist/client/theme-default/components/HoverImage.vue
new file mode 100644
index 0000000..7aa025b
--- /dev/null
+++ b/node_modules/vitepress/dist/client/theme-default/components/HoverImage.vue
@@ -0,0 +1,87 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import type { DefaultTheme } from 'vitepress/theme'
+
+const props = defineProps<{
+  item: DefaultTheme.HoverImage
+}>()
+
+const item = props.item
+
+const shown = ref(false)
+
+const mobileDeviceReg = /android|iphone|ipad|ipod/
+const ua = window?.navigator?.userAgent?.toLowerCase()
+const isMobile = mobileDeviceReg.test(ua)
+
+const onMouseEnter = () => {
+  if (isMobile) return
+  shown.value = true;
+} 
+
+const onMouseLeave = () => {
+  if (isMobile) return
+  shown.value = false
+}
+
+const onClick = () => {
+  if (!isMobile) return
+  window.location.href = item.src
+}
+
+</script>
+
+<template>
+  <div class="hover-image"
+    @mouseenter="onMouseEnter"
+    @mouseleave="onMouseLeave"
+    @click="onClick">
+    <span class="text">{{ item.text }}</span>
+    <img v-show="shown" 
+      class="image"
+      :src="item.src"
+      :alt="item.text"
+    />
+  </div>
+</template>
+
+<style scoped>
+.hover-image {
+  position: relative;
+  padding: 0 12px;
+  .text {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    line-height: var(--vp-nav-height);
+    color: var(--vp-c-text-1);
+    font-size: 14px;
+    font-weight: 500;
+    transition: color 0.25s;
+    cursor: pointer;
+    user-select: none;
+  }
+  .text:hover {
+    color: var(--vp-c-brand-1);
+  }
+  .image {
+    position: absolute;
+    max-width: 200px;
+    width: 200px;
+    height: 200px;
+    top: 100%;
+    right: 0;
+    border-radius: 10px;
+    box-shadow: 1px 1px 10px #ccc;  
+  }
+}
+@media (max-width: 768px) {
+  .hover-image {
+    height: 48px;
+    padding: 0 4px 0 0;
+    border-bottom: 1px solid var(--vp-c-divider);
+    overflow: hidden;
+    transition: border-color 0.5s;
+  }
+}
+</style>
diff --git a/node_modules/vitepress/dist/client/theme-default/components/VPDocOutlineItem.vue b/node_modules/vitepress/dist/client/theme-default/components/VPDocOutlineItem.vue
index 89ea971..6f53dda 100644
--- a/node_modules/vitepress/dist/client/theme-default/components/VPDocOutlineItem.vue
+++ b/node_modules/vitepress/dist/client/theme-default/components/VPDocOutlineItem.vue
@@ -1,13 +1,19 @@
 <script setup lang="ts">
+import { computed } from 'vue'
 import type { MenuItem } from '../composables/outline'
 
-defineProps<{
+const props = defineProps<{
   headers: MenuItem[]
   root?: boolean
 }>()
 
-function onClick({ target: el }: Event) {
-  const id = (el as HTMLAnchorElement).href!.split('#')[1]
+const validHeaders = computed(() => {
+  const headers = props.headers || []
+  return headers.filter(header => !(/^_/g.test(header.title)))
+})
+
+function onClick(link: string) {
+  const id = link.split('#')[1]
   const heading = document.getElementById(decodeURIComponent(id))
   heading?.focus({ preventScroll: true })
 }
@@ -15,8 +21,15 @@ function onClick({ target: el }: Event) {
 
 <template>
   <ul class="VPDocOutlineItem" :class="root ? 'root' : 'nested'">
-    <li v-for="{ children, link, title } in headers">
-      <a class="outline-link" :href="link" @click="onClick" :title="title">{{ title }}</a>
+    <li v-for="{ children, link, title, isSLabel } in validHeaders">
+      <a class="outline-link" :href="link" @click="() => { onClick(link) }" :title="title">
+        <template v-if="isSLabel">
+          <s>{{ title }}</s>
+        </template>
+        <template v-else>
+          {{ title }}
+        </template>
+      </a>
       <template v-if="children?.length">
         <VPDocOutlineItem :headers="children" />
       </template>
diff --git a/node_modules/vitepress/dist/client/theme-default/components/VPNavBarMenu.vue b/node_modules/vitepress/dist/client/theme-default/components/VPNavBarMenu.vue
index 3f8fd8a..41921ae 100644
--- a/node_modules/vitepress/dist/client/theme-default/components/VPNavBarMenu.vue
+++ b/node_modules/vitepress/dist/client/theme-default/components/VPNavBarMenu.vue
@@ -1,8 +1,13 @@
 <script lang="ts" setup>
+import { defineClientComponent } from 'vitepress'
 import { useData } from '../composables/data'
 import VPNavBarMenuLink from './VPNavBarMenuLink.vue'
 import VPNavBarMenuGroup from './VPNavBarMenuGroup.vue'
 
+const HoverImage = defineClientComponent(() => {
+  return import('./HoverImage.vue')
+})
+
 const { theme } = useData()
 </script>
 
@@ -17,6 +22,7 @@ const { theme } = useData()
     </span>
     <template v-for="item in theme.nav" :key="JSON.stringify(item)">
       <VPNavBarMenuLink v-if="'link' in item" :item="item" />
+      <HoverImage v-else-if="'image' === item.customType" :item="item" />
       <component
         v-else-if="'component' in item"
         :is="item.component"
diff --git a/node_modules/vitepress/dist/client/theme-default/components/VPNavScreenMenu.vue b/node_modules/vitepress/dist/client/theme-default/components/VPNavScreenMenu.vue
index d88bb6f..b29fc89 100644
--- a/node_modules/vitepress/dist/client/theme-default/components/VPNavScreenMenu.vue
+++ b/node_modules/vitepress/dist/client/theme-default/components/VPNavScreenMenu.vue
@@ -1,8 +1,13 @@
 <script lang="ts" setup>
+import { defineClientComponent } from 'vitepress'
 import { useData } from '../composables/data'
 import VPNavScreenMenuLink from './VPNavScreenMenuLink.vue'
 import VPNavScreenMenuGroup from './VPNavScreenMenuGroup.vue'
 
+const HoverImage = defineClientComponent(() => {
+  return import('./HoverImage.vue')
+})
+
 const { theme } = useData()
 </script>
 
@@ -10,6 +15,7 @@ const { theme } = useData()
   <nav v-if="theme.nav" class="VPNavScreenMenu">
     <template v-for="item in theme.nav" :key="JSON.stringify(item)">
       <VPNavScreenMenuLink v-if="'link' in item" :item="item" />
+      <HoverImage v-else-if="'image' === item.customType"  :item="item" />
       <component
         v-else-if="'component' in item"
         :is="item.component"
diff --git a/node_modules/vitepress/dist/client/theme-default/components/VPSocialLink.vue b/node_modules/vitepress/dist/client/theme-default/components/VPSocialLink.vue
index 905ec89..c1454a1 100644
--- a/node_modules/vitepress/dist/client/theme-default/components/VPSocialLink.vue
+++ b/node_modules/vitepress/dist/client/theme-default/components/VPSocialLink.vue
@@ -22,7 +22,7 @@ onMounted(async () => {
   ) {
     span.style.setProperty(
       '--icon',
-      `url('https://api.iconify.design/simple-icons/${props.icon}.svg')`
+      ['github', 'npm'].includes(props.icon) ? `url('/icons/${props.icon}.svg')`: `url('https://api.iconify.design/simple-icons/${props.icon}.svg')`
     )
   }
 })
diff --git a/node_modules/vitepress/dist/client/theme-default/composables/outline.js b/node_modules/vitepress/dist/client/theme-default/composables/outline.js
index 6746cc1..26954c0 100644
--- a/node_modules/vitepress/dist/client/theme-default/composables/outline.js
+++ b/node_modules/vitepress/dist/client/theme-default/composables/outline.js
@@ -21,26 +21,32 @@ export function getHeaders(range) {
         const level = Number(el.tagName[1]);
         return {
             element: el,
-            title: serializeHeader(el),
             link: '#' + el.id,
-            level
+            level,
+            ...serializeHeader(el),
         };
     });
     return resolveHeaders(headers, range);
 }
 function serializeHeader(h) {
     let ret = '';
+    let isSLabel = false;
     for (const node of h.childNodes) {
         if (node.nodeType === 1) {
             if (ignoreRE.test(node.className))
                 continue;
+            if (node.nodeName.toLowerCase() === 's') 
+                isSLabel = true
             ret += node.textContent;
         }
         else if (node.nodeType === 3) {
             ret += node.textContent;
         }
     }
-    return ret.trim();
+    return {
+        title: ret.trim(),
+        isSLabel
+    };
 }
 export function resolveHeaders(headers, range) {
     if (range === false) {
diff --git a/node_modules/vitepress/types/default-theme.d.ts b/node_modules/vitepress/types/default-theme.d.ts
index 10681a8..4c82c76 100644
--- a/node_modules/vitepress/types/default-theme.d.ts
+++ b/node_modules/vitepress/types/default-theme.d.ts
@@ -174,6 +174,12 @@ export namespace DefaultTheme {
     props?: Record<string, any>
   }
 
+  export interface HoverImage {
+    text: string;
+    customType: string;
+    src: string;
+  }
+
   export interface NavItemWithLink {
     text: string
     link: string
